Makefile

```diff
@@ -286,13 +286,14 @@ build-rpms-redistributable:
 	hack/build-rpm-release.sh
 .PHONY: build-rpms-redistributable
 
-# Build a release of OpenShift using tito for linux/amd64 and the images that depend on it.
+# Build a release of OpenShift (all the RPMs and images for all platforms)
 #
 # Args:
 #
 # Example:
 #   make release-rpms
-release-rpms: clean build-rpms
+release-rpms: clean build-rpms-redistributable
+	hack/build-base-images.sh
```

Comment:
If @smarterclayton is fine with it, this should replace the previous `make release`